requirements.txt

```diff
@@ -1,16 +1,12 @@
 configobj>=5.0.5
-cherrypy==8.9.1
+cherrypy>=3.6.0,<9.0
 ws4py>=0.3.2
-SQLAlchemy>=1.0.0
+SQLAlchemy>=1.1.0
 six>=1.5.2
 Jinja2>=2.7
 rpctools>=0.3.1
 logging_unterpolation>=0.2.0
 requests>=2.2.1
-pytest>=3.0.1
-mock>=1.0.1,<1.1
-Sphinx>=1.2.1
-coverage>=3.6
```

Comment:
This has been moved into a separate ``test_requirements.txt`` which is a better practice.

This might require a small tweak to the ``ubersystem-deploy`` repo in order to ensure that these dependencies are still installed.  In other words, this *is* a backwards-incompatible change, in that running ``python setup.py develop`` will now install fewer dependencies, and if you want the test dependencies then you'll need to say something like ``pip install -r test_requirements.txt`` (which is what we've started doing in places where we need this).